context/app/static/js/components/Detail/Citation/style.js

```diff
@@ -1,8 +1,14 @@
 import styled from 'styled-components';
 import Paper from '@material-ui/core/Paper';
 
+import { InfoIcon } from 'js/shared-styles/icons';
+
 const FlexPaper = styled(Paper)`
   padding: 30px 40px;
 `;
 
-export { FlexPaper };
+const StyledInfoIcon = styled(InfoIcon)`
+  margin-right: ${(props) => props.theme.spacing(0.5)}px;
```

Comment:
Should this be margin left?

Reply:
yes... but it looks fine without any style at all. Let me take that out...